docs/04-operation-guides/operations/evnt-02-eventing-metrics.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Kyma Eventing Metrics
+---
+
+Kyma Eventing emits several custom metrics which are then visualized using [Grafana Dashboards](./evnt-01-eventing-dashboards.md) to monitor statistics and other information for the Eventing backbone in real time.
+
+### Metrics Emitted by Eventing Publisher Proxy:
+
+| Metric    |  Description |
+|-------------|:--------------|
+| **event_publish_to_messaging_server_errors_total** | Total number of errors while sending events to the messaging server. |
+| **event_publish_to_messaging_server_latency** | Duration of sending events to the messaging server. |
+| **event_type_published** | Total number of events published for a given event type. |
+| **event_requests** | Total number of event requests for a destination service.  |
```

Comment:
I am unable to understand this metric. Add more information or rephrase the sentence?

Reply:
I just used the description from the metric's Help. But I will add more information

Reply:
https://github.com/kyma-project/kyma/blob/9ee7b1f9bcdf6985a93ed10ed945b7a11ff39dd7/components/event-publisher-proxy/pkg/metrics/collector.go#L26